its/src/test/java/com/sonar/it/csharp/CoverageTest.java

```diff
@@ -66,8 +52,8 @@ public void should_not_import_coverage_without_report() throws Exception {
       .doesNotContain("Sensor C# Tests Coverage Report Import")
       .doesNotContain("Coverage Report Statistics:");
 
-    org.sonarqube.ws.Measures.Measure linesToCover = getMeasure("CoverageTest", "lines_to_cover");
-    org.sonarqube.ws.Measures.Measure uncoveredLines = getMeasure("CoverageTest", "uncovered_lines");
+    Measures.Measure linesToCover = getMeasure("CoverageTest", "lines_to_cover");
+    Measures.Measure uncoveredLines = getMeasure("CoverageTest", "uncovered_lines");
```

Comment:
pedantic: you can use `import org.sonarqube.ws.Measures.Measure;` instead of `import org.sonarqube.ws.Measures;` to simplify this even more.